mappings/net/minecraft/client/gui/widget/EntryListWidget.mapping

```diff
@@ -118,7 +118,7 @@ CLASS cwq net/minecraft/client/gui/widget/EntryListWidget
 		ARG 1 renderSelection
 	METHOD setScrollAmount (D)V
 		ARG 1 amount
-	METHOD setSelected (Lcwq$a;)V
+	METHOD setSelected selectEntry (Lcwq$a;)V
```

Comment:
This should not have a name assgined to it.

Reply:
Oops sorry for that